draft-ietf-rtcweb-jsep.xml

```diff
@@ -1140,7 +1140,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTrack" anchor="sec.addTrack">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTrack method adds a MediaStreamTrack to the PeerConnection.
+                    addTrack attempts to minimize the number of transceivers as follows:
+                    The track will be attached to the first compatible transceiver
+                    (of the same media type) which has never had a direction of
+                    "sendonly" or "sendrecv". If no such transceiver exists,
```

Comment:
This *would* have the effect of upgrading a recvonly line to sendrecv if addTrack is called after the initial offer/answer exchange.  

It was a while ago, but I recall that we toyed with the idea of having something like "only reuse the transceiver if it was created by setRemoteDescription(offer)", but there were some edge cases we worked through and decided that this was the most simple approach to take (which we then wrote in the W3C draft).  I don't recall what those were, but I guess we could go dig up old notes if we think it's worth it.